### DISCUSSION TITLE
How can I have the status bar like in the screenshots?

### DISCUSSION THREAD

**Discussion by rjscdev** [BeardedBear/bearded-theme#80] [Q&A] [answered]
I would really like to have the status bar as in the screenshots, what should I do to have it that way? 
![image](https://user-images.githubusercontent.com/88680048/169675658-de538698-d081-4053-8c91-da49027bc064.png)

mine: 
![image](https://user-images.githubusercontent.com/88680048/169675668-bfb0aaec-e7b6-4626-8477-b98f3f2b243b.png)

**Comment by BeardedBear** [comment, marked as answer]
If you talk about the colored errors, i use this extension : https://marketplace.visualstudio.com/items?itemName=bradzacher.vscode-coloured-status-bar-problems :)

**Reply by rjscdev** [reply]
thanks man, also what extension makes that color in the lower left corner?

**Reply by BeardedBear** [reply]
It's just `Remote - WSL` https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl , for connect in my WSL projects ^^

**Reply by rjscdev** [reply]
thanks for the support man, have a nice day pal
